### PR TITLE
update: readme.md

### PR DESCRIPTION
**Pull request by binaryred01**
Please like, comment and subscribe for more pull requests and follow me on twi

### CODE REVIEW

**Review by IndecentDolphin — APPROVED** [review]
Nice one for getting rid of the ghost unity files

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.DS_Store
```
Comment (Danielandro) [inline .gitignore:1]: Nice

File: README.md [hunk 2]
```diff
@@ -1,21 +1,25 @@
 # DragonPirates
 
-Instructions for use:
+## Team
+
```
Comment (Danielandro) [inline README.md:4]: Oh so beautiful

**Review by Danielandro — APPROVED** [review]
Nice Work Squire

**Pull request merged**